content/influxdb/v2.0/query-data/common-queries/count_unique_values_for_column.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Count unique values for a column
+seotitle: Count unique values for a column
+description: >
+  .
+influxdb/v2.0/tags: [queries]
+menu:
+  influxdb_2_0:
+    name: Count unique values
+    parent: Common queries
+weight: 104
+---
+
+Drop all the data except the column you are interested in: in this case, the `location` column.
```

Comment:
This isn't necessary and can be removed. It's a minor optimization, but I think we leave this our for simplicity.